tastytrade/utils.py

```diff
@@ -45,3 +46,11 @@ def validate_response(response: Response) -> None:  # pragma: no cover
                 error_message += f"\n{error['code']}: {error['message']}"
 
         raise TastytradeError(error_message)
+
+
+def object_to_dict(obj):
```

Comment:
Maybe I'm missing where you're using this but I didn't see any usages? Also it may be repeating the functionality of pydantic's `BaseModel.to_dict`.

Reply:
It's a ghost I forgot about. Will roll back the change.

Reply:
I tested with both get_history and get_order_history and made a $0.10 profit in the process, as the complex order doesn't work in the certification environment.

In get_history it is absolutely not visible that the transactions once were part of a complex order.
In get_order_history it is visible, but there is no "dictionary" complex_order, just id's, which are attributes of the class.